inventory/rh-sso/group_vars/identity-hosts.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+rh_sso_host: "192.168.16.3"
+rh_sso_port: 8080
```

Comment:
This comes back to a previous conversation about http/https. But also, this is likely something that should either be part of `all.yml` or just set by default with the ability to override.

Reply:
Yeah, that makes sense.